cardano-config/src/Cardano/TracingOrphanInstances/HardFork.hs

```diff
@@ -232,3 +239,10 @@ instance ToObject (ChainIndepState (BlockProtocol blk))
       => ToObject (WrapChainIndepState blk) where
     toObject verb = toObject verb . unwrapChainIndepState
 
+instance FromJSON PortNumber where
```

Comment:
I don't understand how this can possibly be related to the hard fork infrastructure. Surely this should live in `Cardano.TracingOrphanInstances.Network`.

Reply:
Removed, left here accidentally!